yarn.lock

```diff
@@ -114,6 +114,13 @@
   dependencies:
     "@babel/types" "^7.27.0"
 
+"@babel/runtime@^7.27.0":
```

Comment:
This will be removed. When the library is symlinked to the RN SDK locally, it refuses to run our integration tests without `@babel/runtime` being present. Haven't gotten to properly check why.